profiles/models_test.py

```diff
@@ -0,0 +1,18 @@
+"""
+Model tests
+"""
+# pylint: disable=no-self-use
+from django.test import TestCase
+from profiles.factories import ProfileFactory, UserFactory
+
+
+class ProfileTests(TestCase):
+    """tests for the profile model"""
+
+    def student_id_on_save(self):
+        """test that a student id is set on save"""
+        user = UserFactory.create()
+        profile = ProfileFactory.create(user=user)
+        assert profile.student_id is None
+        profile.save()
```

Comment:
Can you add a test that verifies that the student ids are incremented by one? And a test where `Profile` already has a student id and `save()` is called?